### PR TITLE
Fix libnyx, library upgrades

### PR DESCRIPTION
**Pull request by rmalmain**
trying to debug this:

```
error: failed to get `libnyx` as a dependency of package `libafl_nyx v0.13.0 (/home/runner/work/LibAFL/LibAFL/libafl_nyx)`

Caused by:
  failed to load source for dependency `libnyx`

Caused by:
  Unable to update https://github.com/nyx-fuzz/libnyx.git?rev=6833d23

Caused by:
  failed to clone into: /home/runner/.cargo/git/db/libnyx-86bda2190d84f803

Caused by:
  network failure seems to have happened
  if a proxy or similar is necessary `net.git-fetch-with-cli` may help here
  https://doc.rust-lang.org/cargo/reference/config.html#netgit-fetch-with-cli

Caused by:
  early EOF; class=Net (12); code=Eof (-20)
Error: Process completed with exit code 101.
```

### CODE REVIEW

**Comment by rmalmain** [comment]
It fixes the nyx issue, there is still another problem most likely solved by #2255.

**Comment by domenukk** [comment]
So github's API changed?

**Comment by domenukk** [comment]
Or did some other commit have the same hash? :D

File: fuzzers/tutorial/Cargo.toml [hunk 1]
```diff
@@ -23,7 +23,7 @@ libafl = { path = "../../libafl/", features = ["default", "rand_trait"] }
 libafl_bolts = { path = "../../libafl_bolts/" }
 libafl_targets = { path = "../../libafl_targets/", features = ["sancov_pcguard_hitcounts", "libfuzzer", "sancov_cmplog"] }
 serde = { version = "1.0", default-features = false, features = ["alloc"] } # serialization lib
-lain = { version = "0.5", features = ["serde_support"], git = "https://github.com/Mrmaxmeier/lain.git", rev = "6c8a786" } # We're using a lain fork with updated rand crate
+lain = { version = "0.5", features = ["serde_support"], git = "https://github.com/Mrmaxmeier/lain.git", rev = "f4ae991c30abda5860ff7ac3933d91cc62e9c2a1" } # We're using a lain fork with updated rand crate
```
[the remark this answers is not in the document]
Comment (Mrmaxmeier) [reply fuzzers/tutorial/Cargo.toml]: Opened #2257

**Pull request merged**